sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -433,13 +434,31 @@ protected Symbol visitCast(Cast node, ExpressionAnalysisContext context) {
         }
 
         @Override
+        protected Symbol visitTryCast(TryCast node, ExpressionAnalysisContext context) {
+            DataType returnType = DATA_TYPE_ANALYZER.process(node.getType(), null);
+            if (!CastFunctionResolver.supportsExplicitConversion(returnType)) {
```

Comment:
my mistake i did that ``functionMap.get(returnType) != null`` in supportsExplicitConversion method